src/scribe_data/extract_transform/languages/Bengali/prepositions/query_prepositions.sparql

```diff
@@ -0,0 +1,26 @@
+# tool: scribe-data
+# All Bengali (Q9610) prepositions and their corresponding cases.
+# Enter this query at https://query.wikidata.org/.
+
+SELECT
+  (REPLACE(STR(?lexeme), "http://www.wikidata.org/entity/", "") as ?lexemeID)
+  ?preposition
+  ?case
+
+WHERE {
+  ?lexeme a ontolex:LexicalEntry ;
+    dct:language wd:Q9610 ; # Bengali language
+    wikibase:lexicalCategory wd:Q4833830 ; # Prepositions
+    wikibase:lemma ?lemma .
+
+  # Optional selection of corresponding cases.
```

Comment:
Quick check here, @mhmohona: do Bangla prepositions have cases? So in German if you use the preposition `mit` meaning `with`, afterwards you need to use the `Dativ` case that will determine articles and other forms.

I just checked and it looks like Begali does have nominative and genitive cases, but wanted to check to make sure :) :)

Reply:
Aside from this, looks good! No stress on there not being any data. To be expected in some cases. When I first started Scribe French only had one fully conjugated verb 😊

Reply:
In this case Bangla is a easy language,it doesnt have case like German. 

The hard part of it is verb. For each person, each tense...almost for every thing it has different form of a particular verb.